src/rgw/driver/rados/rgw_d3n_datacache.cc

```diff
@@ -65,6 +65,7 @@ D3nDataCache::D3nDataCache()
 
 void D3nDataCache::init(CephContext *_cct) {
   cct = _cct;
+  // coverity[missing_lock:SUPPRESS]
```

Comment:
this wold be ok if `init()` is only called before any threads access `free_data_cache_size`
@mkogan1 can you please verify?

Reply:
thanks!

Reply:
nvoked from `init_rados()` after ctor